safe.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>SAFENET</title>
+  </head>
+  <body>
+    <!-- top nav bar -->
+    <nav class="top-nav">
+      <ul class="top-ul">
+        <li class="top-li">
+          <a href="#"><i class="fa-brands fa-facebook"></i></a>
+        </li>
+        <li class="top-li">
+          <a href=""><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li class="top-li">
+          <a href="#">English</a>
+        </li>
+        <li class="top-li">
+          <a href="">mypage</a>
+        </li>
+        <li class="top-li">
+          <a href="">logout</a>
+        </li>
+      </ul>
+    </nav>
+    <!-- main navbar  -->
```

Comment:
- [ ] kindly make the mobile popup menu full height so its importance can be felt as we won't be able to navigate when it comes up unless we use it\
![Screenshot (197)_LI](https://user-images.githubusercontent.com/65298048/176853089-31926cc4-b886-420d-a3cd-ca06d4f879fb.jpg)

Reply:
@businesskaniki this requirement is not fulfilled.